src/Aspire.Hosting.MySql/MySqlBuilderExtensions.cs

```diff
@@ -102,14 +103,11 @@ public static IResourceBuilder<T> WithPhpMyAdmin<T>(this IResourceBuilder<T> bui
 
         containerName ??= $"{builder.Resource.Name}-phpmyadmin";
 
-        var configurationTempFileName = Path.GetTempFileName();
```

Comment:
@sebastienros - do we want to bring the other changes outside of EventHubs and ServiceBus to 9.1.1? The DcpExecutor change increases the risk here.

Reply:
mysql containers run as root, but pgweb is non-root, will check if there could be the same issue as EH there. Otherwise I agree we could remove this part.

Reply:
Working on it